applications/StatisticsApplication/README.md

```diff
@@ -0,0 +1,173 @@
+
+Following is an example of integration calculation of non historical velocity. Input variable is node's non-historical container's `VELOCITY` and output variable is same containers `DISPLACEMENT` where integrated value will be stored for each node. The `0` represents echo level for this method object. Blank "" indicates that value method is used.
+
+```python
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.StatisticsApplication as KratosStats
+model = Kratos.Model()
+model_part = model.CreateModelPart("test_model_part")
+sum_method = KratosStats.TemporalMethods.NonHistorical.Nodes.ValueMethods.Sum.Array(model_part, "", Kratos.VELOCITY, 0, Kratos.DISPLACEMENT)
+sum_method.InitializeStatisticsMethod()
+previous_time = 2
+for t in range(3, 6):
+    delta_time = t - previous_time
+    sum_method.CalculateStatistics(delta_time)
+    previous_time = t
+```
+
+#### Mean
+
+In the case of spatial domain, it calculates mean of a given variable for a given container and returns it as shown in following equation. x<sub>i</sub> is the i<sup>th</sup> element's variable value of the corresponding container. Result will have the same type as the type of the variable specified by the user. (If it has higher dimension than a scalar, mean of each dimension will be calculated seperately resulting with a mean having same dimension as the input dimension)
+
+<a href="https://www.codecogs.com/eqnedit.php?latex=\color{White}{\underline{r}&space;=&space;\frac{1}{N}\sum_{i=1}^{N}\underline{x}_i}" target="_blank"><img src="https://latex.codecogs.com/svg.latex?\color{White}{\underline{r}&space;=&space;\frac{1}{N}\sum_{i=1}^{N}\underline{x}_i}" title="\color{White}{\underline{r} = \frac{1}{N}\sum_{i=1}^{N}\underline{x}_i}" /></a>
+
+Following is an example of mean calculation of non historical `VELOCITY` over the whole model part's nodes
+
+```python
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.StatisticsApplication as KratosStats
+model = Kratos.Model()
+model_part = model.CreateModelPart("test_model_part")
+mean = KratosStats.SpatialMethods.NonHistorical.Nodes.ValueMethods.Mean(model_part, Kratos.VELOCITY)
+```
+
+In the case of temporal domain, **Mean** methods is the time integrated quantity's mean for a specific variable. It will be stored each element under user specified variable and a user specified container. x<sub>i</sub> is the i<sup>th</sup> element's variable value of the corresponding container. Result will have the same type as the type of the variable specified by the user preserving the dimensionality as in the spatial case.
```

Comment:
Here again, _time integrated_ is misleading. I would simply call it as the (arithmetic) mean of the (time) series.